ayon_server/settings/enum.py

```diff
@@ -130,6 +142,106 @@ async def anatomy_presets_enum():
     return result
 
 
+def anatomy_template_items_enum(
+    project_name: str | None = None,
+    category: str | None = None
```

Comment:
I think that they should be required...
```suggestion
    project_name: str | None,
    category: Literal["work", "publish", "hero", "delivery", "others", "staging"]
```

Reply:
I feel I used somewhere `render_default` as a template name somewhere, not sure if there is use case to not requiring `category`. @jakubjezek001

Reply:
> I feel I used somewhere render_default as a template name once, not sure if there is use case to not requiring category.

`render_default` is not a thing, `publish_render` might be, but you should use only `render` in that case, the `publish_` prefix is actually trimmed out as far as I know and is not used for more than a year (I hope...).